.github/workflows/auto-assign-to-project.yml

```diff
@@ -0,0 +1,26 @@
+name: Auto Assign to Project(s)
+
+on:
+  pull_request:
+    types: [opened, labeled]
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  assign_one_project:
+    runs-on: ubuntu-latest
+    name: Assign to One Project
```

Comment:
```suggestion
    name: Assign to Project
```

Reply:
Thank you @mondeja for the feedback! I just made some changes